### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by ravikumar1002**
* Create new playlist 
 * Add video in playlist from video listing page
 * Playlists Page to show all Playlists and their video count
 * Delete Playlist
 * Delete video in single playlist page

### CODE REVIEW

File: src/context/auth-context.jsx [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
 
+    // useEffect(() => {
+    //     if(localStrogeItem?.token){
+    //         (async() => {
+    //             const data = await signupHandler(localStrogeItem.user.email, localStrogeItem.user.password)
+    //             setUser(data)
+    //         })()
+    //     }
+    // },[localStrogeItem?.token === ])
+
```
Comment (romabulani) [inline src/context/auth-context.jsx]: You can remove the comments if not required

**Review by romabulani — APPROVED** [review]
- Clean and modular
- good to merge

File: src/context/auth-context.jsx (hunk 1 above)
Comment (ravikumar1002) [reply src/context/auth-context.jsx]: Done

File: src/hooks/body-scrolling.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react"
+const useImperativeDisableScroll = ({ element, disabled }) => {
```
Comment (vinc3nati) [inline src/hooks/body-scrolling.jsx:2]: nice to see use of custom hook for body scrolling

**Review by vinc3nati — APPROVED** [review]
Nice UI, great modular structure

**Pull request merged**